pkgs/development/python-modules/markdown-it-py/default.nix

```
@@ -85,7 +85,7 @@ buildPythonPackage rec {
   meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242